metamorph/src/main/java/org/metafacture/metamorph/Metamorph.java

```diff
@@ -64,6 +65,7 @@
 @FluxCommand("morph")
 public final class Metamorph implements StreamPipe<StreamReceiver>, NamedValuePipe, Maps {
 
+    private static final String ELSE_AND_PASS_ENTITY_EVENTS_KEYWORD = "_elseAndPassEntityEvents";
```

Comment:
Could it make sense to have two new names to make the difference clear (and keeping `_else` for compatibility)?

Something like `_elseFlat` and `_elseFull` (as in 'full structure' or 'full data').

Reply:
`_elseFlat` / `_elseNested`? `_elseFlattened` / `_elseUnflattened`?

Reply:
Ok, two then ... Then I like `_elseFlattened` / `_elseNested` most because the last one ("nested") is more self-explaining than `entities` for users not thinking in metafacture terms, and `_elseFull` is not that clear what is meant by that. I would prefer `Flattened` more than `flat` because of the semantics (see https://www.merriam-webster.com/thesaurus/flat where one antonym of `flat` is `inexact) where `flattened` is more focused on the morphological structure.
Also of symmetric reasons (both words ends on `ed`).
(thumbs up when you agree).